src/components/Tabs/Tabs.tsx

```diff
@@ -65,29 +65,20 @@ export const Tabs = ({
   onChange,
   ...other
 }: Props) => {
-  /**
-   * Initialize states, constants, and refs
-   */
   const ref = useRef<number | undefined>();
   const headerRef = useRef<HTMLDivElement>(null);
   const [activeIndexState, setActiveIndexState] = useState(activeIndex);
   const [scrollableLeft, setScrollableLeft] = useState<boolean>(false);
   const [scrollableRight, setScrollableRight] = useState<boolean>(false);
 
-  /**
-   * Set the only children components allowed within <Tabs> to be Tab
-   */
-  const tabs = useCallback<() => React.ReactElement<TabProps>[]>(() => {
-    // TODO: apply types from module, specify type definitions in custom.d.ts,
-    // or replace with inline equivalent
+  /** Get all children that are Tab components. Ignore any others. */
+  const getTabs = useCallback(() => {
     return allByType(children, Tab);
   }, [children]);
 
-  const tabRefs = tabs().map(() => React.createRef<HTMLAnchorElement>());
-
-  // we can't use the hook in an iterator like this, so generate the base and increment if needed
-  const [idVar, setId] = useState<string[]>([]);
-  const [ariaLabelledByVar, setAriaLabelledBy] = useState<string[]>([]);
+  const tabRefs = getTabs().map(() => React.createRef<HTMLAnchorElement>());
+  const [tabPanelIds, setTabPanelIds] = useState<string[]>([]);
```

Comment:
Probably don't need an id for each tab panel. Should be able to use a single id, since only one is rendered at a time.